app/views/activities/_filters.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="activity-filters ">
+  <%= form_with url: project_activities_path(current_project.id), method: :get, remote: true, class: 'form-row d-flex flex-row' do |f| %>
+    <%= f.select :user, options_from_collection_for_select(User.all, :id, :name, params[:user]), { include_blank: 'Filter by user'}, { class: 'form-select m-2 form-select-sm', onchange: '$(this.form).submit();' } %>
```

Comment:
- This `<select>` is missing a `<label>`
- Let's change the include blank to `All users` since the label will say something like `User`
- Let's avoid inline `onchange` attributes. I believe this is inconsistent with the rest of the app. Setting up a listener in a JS file for all elements that need it will be cleaner since they all call the same method on change.

Reply:
1. I am a little skeptical about the label here as it will affect the display. That's why I added the `Filter by user` in the `include blank` as a CTA. I tried to be careful with spacing. How do you feel about adding hidden labels?
2. I agree,`All users` is much better.
3. Agree